docs/en_US/Compression/QuantizationSpeedup.rst

```diff
@@ -0,0 +1,141 @@
+Speed up Mixed Precision Quantization Model
+===========================================
+
+*This feature is in Beta version.*
+
+Introduction
+------------
+
+Deep learning network has been computationly intensive and memory intensive 
+which increase the difficulty of deploying deep neural network model. Quantization is a 
+fundamental technology which is widely used to reduce memory footprint and speed up inference 
+process. Many frameworks begin to support quantization, but few of them support mixed precision 
+quantiation. Frameworks like haq only support simulated mixed precision quantization which will 
+not speed up the inference process. To get real speed up of mixed precision quantization and 
+help people get the real feedback from hardware, we design a tool which combine nni and tensorrt together.
+
+
+Design and Implementation
+-------------------------
+
+To support speeding up mixed precision quantization, we divide framework into two part, frontend and backend.  
+Frontend could be popular raining framework such as pytorch, tensorflow etc. Backend could be inference 
+framework for different hardware, such as tensorrt. At present, we support pytorch as frontend and 
+tensorrt as backend. To convert pytorch model to tensorrt engine, we leverage onnx as intermediate graph 
+representation. In this way, we convert pytorch model to onnx model, then tensorrt parse onnx 
+model to generate inference engine. 
+
+
+For the purpose of getting mixed precision engine, we need to provide a pytorch model, calibrate dataset 
+and corresponded bit config. The calibrate set is used to calibrate quantized module. What' s more, the 
+bit config a dictionary whose key is layer name and value is bit width. After getting mixed precision engine, 
+we can do inference with given input data.
+
+Prerequisite
+------------
+CUDA version >= 11.0
+
+TensorRt version >= 7.2.1.6
```

Comment:
7.2.1.6 is strange

Reply:
TensorRT version >= 7.2 instread. Have tested.